models/glif_psc_double_alpha.h

```diff
@@ -0,0 +1,487 @@
+asc_amps                   double vector  After-spike current amplitudes in
+                                          pA (deltaIj in Equation (7) in [1]_)
+asc_r                      double vector  Current fraction following spike
+                                          coefficients for fj in Equation (7)
+                                          in [1]_
+th_voltage_index           double         Adaptation index of threshold - A
+                                          'leak-conductance' for the
+                                          voltage-dependent component of the
+                                          threshold in 1/ms (av in Equation
+                                          (4) in [1]_)
+th_voltage_decay           double         Voltage-induced threshold time
+                                          constant - Inverse of which is the
+                                          time constant of the
+                                          voltage-dependent component of the
+                                          threshold in 1/ms (bv in Equation
+                                          (4) in [1]_)
+tau_syn_fast               double vector  Time constants of the faster
+                                          synaptic alpha function in ms
+tau_syn_slow               double vector  Time constants of the slower
+                                          synaptic alpha function in ms
+amp_slow                   double vector  Relative amplitude of the slower
+                                          synaptic alpha function
+E_rev                      double vector  Reversal potential in mV
+spike_dependent_threshold  bool           flag whether the neuron has
+                                          biologically defined reset rules
+                                          with a spike dependent threshold
+                                          component
+after_spike_currents       bool           flag whether the neuron has after
+                                          spike currents
+adapting_threshold         bool           flag whether the neuron has a
+                                          voltage dependent threshold component
+========================= =============== =====================================
+
+References
+++++++++++
+
+.. [1] Teeter C, Iyer R, Menon V, Gouwens N, Feng D, Berg J, Szafer A,
+       Cain N, Zeng H, Hawrylycz M, Koch C, & Mihalas S (2018)
+       Generalized leaky integrate-and-fire models classify multiple neuron
+       types. Nature Communications 9:709.
+.. [2] Meffin, H., Burkitt, A. N., & Grayden, D. B. (2004). An analytical
+       model for the large, fluctuating synaptic conductance state typical of
+       neocortical neurons in vivo. J.  Comput. Neurosci., 16, 159-175.
+
+See also
+++++++++
+
+gif_psc_exp_multisynapse, gif_cond_exp, gif_cond_exp_multisynapse, gif_pop_psc_exp,
+glif_psc
```

Comment:
Can you also add a reference to ``glif_psc_double_alpha`` in the See also section of ``glif_psc.h``?

Reply:
I added the reference to 'glif_psc.h'.